FAQ.md

```diff
@@ -105,19 +102,18 @@ Once that is done and the domain is marked as available, and you've tested with
 
 You will need to do the following in quick succession:
 
-- Change the DNS record (the records will all have the same destination as the sub domain you created for testing in step 5 above)
-- Remove the CNAME from your Cloudfront distribution.
+- Change the DNS record (the records will all have the same destination as the subdomain you created for testing in step 6 above)
```

Comment:
I think this should actually be step 5? (looks like should've referred to step 4 in prev version)